README.md

```diff
@@ -27,11 +27,11 @@ The following table shows the status of various features.
 
 ## Requirements
 
-**Java 11 or 17 provided by OpenJDK or Oracle**. Eclipse OpenJ9 is not
+**Java 8, 11, 17 provided by OpenJDK or Oracle**. Eclipse OpenJ9 is not
```

Comment:
Let's discourage Java 8, we might have forgot to remove it in the docs and only removed here.

Reply:
Fixed!